### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by dnr36**

### CODE REVIEW

File: Group Projects/group-6/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 
 # Group #6 
-# Project Pitch
+# Project Pitch: Using childcare as incentive for women entrpreneurs to formalize
```
Comment (genjuro86) [inline Group Projects/group-6/README.md:3]: Yes, we should have put a title on the pitch. Good job.

**Review by genjuro86 — APPROVED** [review]

**Review by bnhippolyte — CHANGES_REQUESTED** [review]
There's a spelling error in entrepeneur

**Review by bnhippolyte — CHANGES_REQUESTED** [review]
Should we say "as an incentive" as opposed to "as incentive "

**Review by bnhippolyte — APPROVED** [review]
Looks great !

**Pull request merged**